toolbox/src/common.rs

```diff
@@ -19,11 +17,15 @@ pub const TNSR_DISCOUNT_BPS: u64 = 2500;
 pub const TAKER_FEE_BPS: u64 = 200;
 pub const MAKER_BROKER_PCT: u64 = 80; // Out of 100
 
+pub const fn pubkey(base58str: &str) -> Pubkey {
+    Pubkey::new_from_array(five8_const::decode_32_const(base58str))
```

Comment:
I get that `five8_const` is faster... but should we go with a more popular base58 package? https://crates.io/crates/bs58

Eg 0-days will be circulated quicker for popular packages than less popular ones.

Reply:
It's not about speed so much as I don't know how to use `bs58` to create this `const fn` helper. It has a `into_array_const_unwrap` function but that expects the correct number of bytes passed in so it doesn't panic if you try to create a pubkey from a string like `"123"`. There's a way to do this with a proc macro but [it's a lot more complicated](https://github.com/tensor-foundation/toolbox/pull/26).

Another thing about `five8_const` is that it only relies on `five8_core` which itself has zero dependencies.

Reply:
cc @paul-merugu any objections here.

Reply:
None here @richardwu .  reviewed the package. It seems skinny overall + zero dependencies is also a good thing in this context.